plugins/evaluation-plugin/src/com/intellij/cce/interpreter/DelegationCompletionInvoker.kt

```diff
@@ -46,6 +46,16 @@ class DelegationCompletionInvoker(private val invoker: CompletionInvoker, projec
     }
   }
 
+  override fun callImportCompletion(expectedText: String): com.intellij.cce.core.Lookup {
+    var result: com.intellij.cce.core.Lookup? = null
+    ApplicationManager.getApplication().executeOnPooledThread {
+      ApplicationManager.getApplication().runReadAction {
```

Comment:
let's move this logic to `JavaImportListProvider`. Theoretically, auto import in different languages might require different thread conditions.